mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MailboxMapperACLTest.java

```diff
@@ -0,0 +1,260 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.store.mail.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.mailbox.model.MailboxACL;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.model.SimpleMailboxACL;
+import org.apache.james.mailbox.store.mail.MailboxMapper;
+import org.apache.james.mailbox.store.mail.model.impl.SimpleMailbox;
+import org.junit.Assume;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Generic purpose tests for your implementation MailboxMapper.
+ * 
+ * You then just need to instantiate your mailbox mapper and an IdGenerator.
+ */
+public abstract class MailboxMapperACLTest {
+    private final static long UID_VALIDITY = 42;
+    public static final boolean POSITIVE = true;
+    public static final boolean NEGATIVE = !POSITIVE;
+
+    private MailboxPath benwaInboxPath;
+    private Mailbox benwaInboxMailbox;
+
+    @Rule
+    public ExpectedException expected = ExpectedException.none();
+    private MailboxMapper mailboxMapper;
+    private MapperProvider mapperProvider;
+
+    protected abstract MapperProvider createMapperProvider();
+
+    public void setUp() throws Exception {
+        this.mapperProvider = createMapperProvider();
+        Assume.assumeTrue(mapperProvider.getSupportedCapabilities().contains(MapperProvider.Capabilities.MAILBOX));
+        Assume.assumeTrue(mapperProvider.getSupportedCapabilities().contains(MapperProvider.Capabilities.ACL_STORAGE));
+
+        this.mailboxMapper = mapperProvider.createMailboxMapper();
+
+
```

Comment:
extra line

Reply:
Already spotted by @rouazana  and corrected.